src/components/Meeting/EditMeeting.jsx

```diff
@@ -0,0 +1,64 @@
+import { useContext, useState } from 'react';
+import { hostUrl } from '../../utils/urls';
+import { MEETING_FIELDS } from '../../common/fields';
+import InputFormRow from '../../common/InputFormRow';
+import FormSubmitButton from '../../common/FormSubmitButton';
+import { UserContext } from '../../context/UserContext';
+import { useNavigate } from 'react-router-dom';
+import { checkTextForProfanity } from '@/common/profanity';
+
+export default function EditMeeting() {
+    const params = new URLSearchParams(window.location.search);
+    const meetingId = params.get('meetingId');
+    const [meetingInfo, setMeetingInfo] = useState({});
+    const { user } = useContext(UserContext);
+
+    const navigate = useNavigate()
+
+    const postMeeting = () => {
```

Comment:
The name of the function is postMeeting, but I can see there is method PATCH for the fetch